### DISCUSSION TITLE
For how long will V5 be maintained?

### DISCUSSION THREAD

**Discussion by williamrjribeiro** [remix-run/react-router#8398] [Q&A]
Hi there! 👋 

Maybe this question has been answered before but I cannot find it but ... **For how long will V5 be maintained?**

I'm just trying to figure out how soon should we start migrating to V6.

Cheers!

**Comment by timdorr** [comment]
We'll release any PRs that get merged in or fix any major issues with it, but it should be considered "finished" and you should migrate to v6 whenever you're able to.
